src/main.ts

```diff
@@ -226,17 +229,40 @@ export class Purchases {
    * package from {@link Purchases.getOfferings}. This method will present the purchase
    * form on your site, using the given HTML element as the mount point, if
    * provided, or as a modal if not.
+   * @deprecated - please use .purchase
```

Comment:
@tonidero is this enough to mark the endpoint as deprecated?

Should we update the documentation somewhere else?

Reply:
I believe it is yes: https://tsdoc.org/pages/tags/deprecated/. You can also generate the docs locally and see how they look, to make sure it's marked as deprecated.

We should update the docs in docusaurus with the new way to purchase once this is merged

Reply:
Creating a ticket

Reply:
Done: https://linear.app/revenuecat/issue/BIL-590/[trials]-update-docusaurus-docs-with-the-new-purchase-method